dev/config/pyramid_oereb.yml.mako

```diff
@@ -238,13 +238,16 @@ pyramid_oereb:
         fr: https://wms.geo.admin.ch/?SERVICE=WMS&REQUEST=GetMap&VERSION=1.3.0&STYLES=default&CRS=EPSG:2056&BBOX=2475000,1065000,2850000,1300000&WIDTH=493&HEIGHT=280&FORMAT=image/png&LAYERS=ch.swisstopo-vd.amtliche-vermessung
       layer_index: 0
       layer_opacity: 1.0
+      verify_certificate: True
```

Comment:
Add "# Option to check certificate for external WMS. Default and recommended setting: True"

Reply:
The comment is already there, subsequently to review by Anne

Reply:
Thanks, copied comment here as well